packages/react-utilities/src/compose/getSlots.test.tsx

```diff
@@ -51,7 +51,7 @@ describe('getSlots', () => {
   it('returns a component slot with no children', () => {
     type Slots = {
       root: IntrinsicShorthandProps<'div'>;
-      icon: ObjectShorthandProps<FooProps>;
+      icon: ComponentSlotProps<React.ComponentType<FooProps>>;
```

Comment:
Does this work?
```suggestion
      icon: ComponentSlotProps<typeof Foo>;
```

Reply:
Oh yeah, that totally works, I didn't realize there was a `Foo` component already.